keyboards/mwstudio/mmk_3/info.json

```
@@ -0,0 +1,55 @@
+{
+    "manufacturer": "MWStudio",
+    "keyboard_name": "MMK_3",
+    "maintainer": "TW59420",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["A3", "A4", "A5"],
+        "rows": ["A6"]
+    },
+    "processor": "STM32F103",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "brightness_steps": 17,
+        "hue_steps": 10,
+        "led_count": 5,
+        "saturation_steps": 17
```

Should avoid re-specifying defaults
```suggestion
        "hue_steps": 10,
        "led_count": 5
```